mailet/api/src/main/java/org/apache/mailet/PerRecipientHeaders.java

```diff
@@ -31,7 +32,7 @@
 import com.google.common.collect.ArrayListMultimap;
 import com.google.common.collect.Multimap;
 
-public class PerRecipientHeaders {
+public class PerRecipientHeaders implements Serializable {
```

Comment:
does it fix something ?

Reply:
Yes. see later on. (File implementations)

I choose to move these fix in separate commits. For readability. Should I incorporate them in the related test commit?